### PR TITLE
ProgressIndicator: improve high contrast colors for progress bar

### LINKED ISSUE #10849: ProgressIndicator: current position not clearly visible in high contrast.
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Bug1 of #8388: Current position in progress Indicator is not that much prominent and clear in High contrast modes.

### Please provide a reproduction of the issue in a codepen:
image

#### Actual behavior:

The current progress flashes by in white on a white bar so the lack of contrast makes it not very visible.

#### Expected behavior:

Should be blue or some other color in HC mode to make it easy to distinguish.

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #10849

#### Description of changes

Update high contrast colors to make progress easier to see.

progress

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11451)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
snaps...grrrr....thought it was just line endings. i'll rebuild and push again

File: packages/office-ui-fabric-react/src/components/__snapshots__/Tooltip.Overflow.Example.tsx.shot [hunk 1]
```diff
@@ -115,7 +115,7 @@ exports[`Component Examples renders Tooltip.Overflow.Example.tsx correctly 1`] =
         role="switch"
         type="button"
       >
-        <div
+        <span
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/__snapshots__/Tooltip.Overflow.Example.tsx.shot]: no clue

Comment (ecraig12345) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Tooltip.Overflow.Example.tsx.shot]: Oh that's my PR #10901 conflicting with yours #11445...I added a toggle to an example and you changed the toggle styles. I fixed it in #11452.

Comment (ecraig12345) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Tooltip.Overflow.Example.tsx.shot]: I mean I opened the fix...please feel free to approve and merge. :)

File: packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.styles.ts [hunk 2]
```diff
@@ -107,7 +107,12 @@ export const getStyles = (props: IProgressIndicatorStyleProps): IProgressIndicat
             position: 'absolute',
             minWidth: '33%',
             background: `linear-gradient(to right, ${progressTrackColor} 0%, ${palette.themePrimary} 50%, ${progressTrackColor} 100%)`,
-            animation: `${isRTL ? IndeterminateProgressRTL : IndeterminateProgress} 3s infinite`
+            animation: `${isRTL ? IndeterminateProgressRTL : IndeterminateProgress} 3s infinite`,
+            selectors: {
+              [HighContrastSelector]: {
```
Comment (xugao) [inline packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.styles.ts]: I dont think we need this here? having background color set above should cover this case.

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.styles.ts]: the existing gradient looks pretty bad in high contrast. this feels much more intentional than an oddly rendered gradient. It doesn't use a system color, but the actual primary blue. And the gradient chops off quite quickly
 
progress2

**Pull request merged**